arbitrator/config/arbitrator_params.yaml

```diff
@@ -26,4 +26,4 @@ use_fixed_costs: true
 # Map: The priorities associated with each plugin during the planning 
 # process, values will be normalized at runtime and inverted into costs
 # Unit: N/a
-plugin_priorities: {RouteFollowingPlugin: 4.0, WorkZonePlugin: 8.0, PlatooningStrategicPlugin: 1.0}
+plugin_priorities: {SCIStrategicPlugin: 2.0, RouteFollowingPlugin: 1.0, WorkZonePlugin: 1.0, PlatooningStrategicPlugin: 1.0}
```

Comment:
SCI is short for stop-controlled intersection. I used the acronym mainly for the namespace and class names to avoid long lines. But I can change the plugin name if needed.

Reply:
More I'm just wondering if there is an industry standard name for this operation, if not SCI is okay, but also a common acronym in the gov't space, so might be a bit confusing. If SCI is the industry standard name that's just fine though, just thought it might be wise to check with the transportation engineers to see if they had any input.